kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx

```diff
@@ -137,9 +137,13 @@ const Details: React.FC = () => {
       <Table isFullwidth>
         <thead>
           <tr>
-            <TableHeaderCell> </TableHeaderCell>
-            <TableHeaderCell title="Topic" />
-            <TableHeaderCell title="Messages behind" />
+            <S.TableHeaderConsumerCell size="small">
+              Topic
+            </S.TableHeaderConsumerCell>
+            <S.TableHeaderConsumerCell> </S.TableHeaderConsumerCell>
+            <S.TableHeaderConsumerCell size="large">
+              Messages behind
+            </S.TableHeaderConsumerCell>
```

Comment:
```suggestion
            <TableHeaderCell colSpan={2} title="Topic"/>
            <TableHeaderCell title="Messages behind" />
```

Reply:
Also you need to play with ToggleButton & TableKeyLink components in ListItem.

Reply:
Also Important to update all the tables where we use same approach

Reply:
done